image/types/Cargo.toml

```diff
@@ -9,6 +9,7 @@ edition = "2021"
 doctest = false
 
 [dependencies]
+arbitrary = { version = "1.3.0", optional = true, features = ["derive"] }
```

Comment:
To make it easier to keep all the versions the same, can we please add this as a workspace dependency?

https://doc.rust-lang.org/cargo/reference/workspaces.html#the-dependencies-table

Reply:
You should still be able to use optional with workspace deps. For example:

https://github.com/chipsalliance/caliptra-sw/blob/2ea7949c437c37541b3dc41cd046cfd60f54c701/hw-model/Cargo.toml#L24

Reply:
This doesn't work (defined `workspace.dependencies` in fuzz/Cargo.toml). It seems this crate's dependencies are still used from inside the actual root workspace, not the local workspace.

Reply:
Yeah, I was just referring to the image/types Cargo.toml (and any other crates in the main workspace). For the fuzz workspace you can do whatever you need.

Reply:
Ah, I understand now. So, we'll define the version once for the root workspace, and also for the targets. Okay, then this is done now.